cranelift/codegen/src/machinst/buffer.rs

```diff
@@ -2494,10 +2546,14 @@ mod test {
             1,
         );
         buf.put1(4);
+        buf.bind_label(label(1), ctrl_plane);
+        buf.put1(0xff);
+        buf.bind_label(label(2), ctrl_plane);
```

Comment:
Looks like line 2525 needs to be changed to `buf.reserve_labels_for_blocks(3);`.

Reply:
Yep, that explains the failures in CI, thanks!